### DISCUSSION TITLE
Relative links break after quotes

### DISCUSSION THREAD

**Discussion by Pixelrella** [iamjackg/md2cf#66] [Q&A] [answered]
Hi,

I am using md2cf to build company internal documentation. I was very happy with relative links being released in the last version. I noticed though that they break after making a quote with `>`. 

Before and in the quote, the links are working, after that if will replace all links with the same page just adding an int that is counting up. It seems to use the first page that was linked.

Anything I can do on my end to fix this?
I wouldn't want to remove the quotes as we are using them for callouts.

Thanks in advance!

**Example**:
```md
[First link to relative page](./first/link/to/relative/page)

> **Tip:** This is a callout with a [second link to a relative page](./second/link/to/relative/page)

[Third link to relative page](./third/link/to/relative/page)
[Forth link to relative page](./forth/link/to/relative/page)
```

*Working*:
[First link to relative page] will be rendered with (./first/link/to/relative/page+0)
[second link to a relative page] will be rendered with (./second/link/to/relative/page)

*Broken*:
[Third link to relative page] will be rendered with ./first/link/to/relative/page+1
[Forth link to relative page] will be rendered with ./first/link/to/relative/page+2

*Expected*:
[Third link to relative page] will be rendered with ./third/link/to/relative/page
[Forth link to relative page] will be rendered with ./forth/link/to/relative/page

**Comment by Pixelrella** [comment]
I dug into this a little more and found out that using quote is unrelated. It simply does not do more than nine links properly. The tenth link and all after are replace with ./first/link/to/relative/page+<n>.

**Comment by iamjackg** [comment]
Thanks for reporting this: I'll take a look!

**Comment by iamjackg** [comment]
I can't reproduce this. I created files from `1.md` to `20.md`, then created a file that references them all, and they're all rendered correctly.

Could you provide me with a complete minimal example that shows the bug on your side, please?

**Comment by Pixelrella** [comment]
Sure, here you go.

I have three pages in a folder:

```
Test
∟ TestOne.md
∟ TestTwo.md
∟ TestThree.md
```

This is what TestOne.md looks like:
```markdown
# Test One

## First links

1. [Test Two](TestTwo.md)
2. [Test Two](TestTwo.md)
3. [Test Two](TestTwo.md)
4. [Test Two](TestTwo.md)
5. [Test Two](TestTwo.md)
6. [Test Two](TestTwo.md)
7. [Test Two](TestTwo.md)
8. [Test Two](TestTwo.md)

## Next links

1. [Test Three](TestThree.md)
2. [Test Three](TestThree.md)
3. [Test Three](TestThree.md)
4. [Test Three](TestThree.md)
5. [Test Three](TestThree.md)
6. [Test Three](TestThree.md)

```

This is what the generated xhtml looks like:

```xhtml
<h1>Test One</h1>\n

<h2>First links</h2>\n

<ol>\n

    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two\">Test Two</a></li>\n
</ol>\n

<h2>Next links</h2>\n

<ol>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2645458954/Test+Three\">Test Three</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two0\">Test Three</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two1\">Test Three</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two2\">Test Three</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two3\">Test Three</a></li>\n
    <li><a href=\"https://company.atlassian.net/wiki/spaces/TE/pages/2644639775/Test+Two4\">Test Three</a></li>\n
</ol>"
```

Instead of pointing to Page Test Three, the links point to `/Test+Two<n>\`.

**Reply by iamjackg** [reply]
Thanks, this is super helpful. I found the bug, which is honestly a pretty silly one :P I'll fix it right away!

**Comment by iamjackg** [comment, marked as answer]
This should be fixed by #67 in [version 2.0.2](https://github.com/iamjackg/md2cf/releases/tag/v2.0.2)

**Reply by Pixelrella** [reply]
Yay! Great :) Thank you.
